docs_src/microservices/core/data/Ch-CoreData.md

```diff
@@ -77,3 +77,26 @@ The two following High Level Interaction Diagrams show:
 **Core Data Request Event Reading or Data for a Device**
 
 ![image](EdgeX_CoreDataEventReading.png)
+
+## Configuration Properties
+
+Please refer to the general Configuration documentation for configuration properties common across all services.
+
+In order to support publishing events via message bus, Core-Data has the following additional configuration section. Changes made to any of these properties while the service is running will not be reflected until the service is restarted.
+
+|Configuration  |     Default Value     |             Dependencies|
+| --- | --- | -- |
+| **Entries in the MessageQueue section of the configuration allow for publication of events to a message bus** |
+|MessageQueue Protocol | tcp | Indicates the connectivity protocol to use to use the bus.|
+|MessageQueue Host | * | Indicates the host of the messaging broker, if applicable.|
+|MessageQueue Port | 5563 | Indicates the port to use when publishing a message.|
+|MessageQueue Type | zero | Indicates the type of messaging library to use. Currently this is ZeroMQ by default. Refer to the `go-mod-messaging` module for more information. |
```

Comment:
Should we list the `mqtt` & `redisstreams` as options here so don't have to jump the go-mod-messaging to find them?

Reply:
I'd prefer to not do that here. Any supporting documentation for the full range of configuration options as applicable to the various providers supported by any module should be found in that module's ReadMe IMHO. I do see that I did not actually link the reference to `go-mod-messaging` so I'll make that change to make cross-referencing easier.